### PR TITLE
Alters immovable rod announcement text

### PR DESCRIPTION
**Pull request by Bawhoppen**
:cl: Flavor and Immersion Committee
tweak: Immovable Rod warning announcement is no longer entirely as immersion breaking.
/:cl:

~~I know it's a piece of our heritage... but times change, and I think we all have to accept that the previous announcement is a little bit too silly and immersion breaking.~~

~~(Sorry to all who this change saddens)~~

Should be *slightly* more immersive now, without anything being sacrificed.

### CODE REVIEW

File: code/modules/events/immovable_rod.dm [hunk 1]
```diff
@@ -27,7 +27,7 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 	announceWhen = 5
 
 /datum/round_event/immovable_rod/announce(fake)
-	priority_announce("What the fuck was that?!", "General Alert")
+	priority_announce("Unknown object detected on collision course with the station.", "General Alert")
```
Comment (Bawhoppen) [inline code/modules/events/immovable_rod.dm]: If anyone has a better suggestion for specific text, feel free of course

**Comment by TheVekter** [comment]
Booooooooo. I'm all for IMMERSHUN but this is just splitting hairs.

File: code/modules/events/immovable_rod.dm (hunk 1 above)
Comment (TheVekter) [reply code/modules/events/immovable_rod.dm]: "Unknown object detected on collision co-" CLANG "What the fuck was that?!"

Comment (Bawhoppen) [reply code/modules/events/immovable_rod.dm]: something like that could definitely work, we can brainstorm more ideas for the absolute best version

**Comment by cacogen** [comment]
what

**Comment by Bawhoppen** [comment]
Okay, I have edited it to allow for both immersion, and maintaining the reference.

File: code/modules/events/immovable_rod.dm [hunk 2]
```diff
@@ -27,7 +27,7 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 	announceWhen = 5
 
 /datum/round_event/immovable_rod/announce(fake)
-	priority_announce("What the fuck was that?!", "General Alert")
+	priority_announce("Unknown high-density object detected on collision co-- What the fuck was that?!", "General Alert")
```
Comment (Bawhoppen) [inline code/modules/events/immovable_rod.dm]: I want to play with the formatting for this a little stille, probably italicizing the "what the fuck" part of it for emphasize, though I don't yet know how to make that work correctly

Any other suggestions for wording, are gladly welcome still.

**Comment by HomocidlePancakes** [comment]
Immersion can not exist as long as we have Ronald McDonald, Marisa Kirasame, Julius Ceaser and such running around

**Comment by optimumtact** [comment]
What the *FUCK* is this?

**Pull request closed without merging**